react_app/src/threeVisualizer.js

```diff
@@ -0,0 +1,230 @@
+import * as THREE from 'three';
```

Comment:
The module name should be lowerCamelCase:
https://google.github.io/styleguide/jsguide.html#es-module-imports

Reply:
Fixed!